.pre-commit-config.yaml

```diff
@@ -25,3 +25,9 @@ repos:
   hooks:
   - id: jupytext
     args: [--sync]
+
+- repo: https://github.com/asottile/pyupgrade
+  rev: v2.32.1
+  hooks:
+  - id: pyupgrade
+    args: ["--py37-plus", "--keep-runtime-typing"]
```

Comment:
As mentioned before, I think we can remove this pre-commit hook. I was looking through the docs to see if there's any way to mark a hook as optional or non-default in the yaml file, but I didn't find anything

Reply:
I have removed the hook now, so we should be ready to go 🚀